unittests/InstructionCountCI/FlagM/VEX_map2.json

```diff
@@ -494,18 +494,23 @@
         "mrs x23, nzcv",
         "cmp x21, #0x1f (31)",
         "csel w4, w20, w22, hi",
-        "and w20, w23, #0xefffffff",
```

Comment:
Interesting little wiggles here. Something to remember to come back to later.

Reply:
It's now later, deleted half of bzhi.